addons/web/tests/test_js.py

```diff
@@ -45,4 +59,4 @@ class MobileWebSuite(odoo.tests.HttpCase):
 
     def test_mobile_js(self):
         # webclient mobile test suite
-        self.browser_js('/web/tests/mobile?mod=web', "", "", login='admin', timeout=1800, failure_message=' tests failed.')
+        self.browser_js('/web/tests/mobile?mod=web', "", "", login='admin', timeout=1800, failure_message=' tests failed.', error_checker=qunit_error_checker)
```

Comment:
`failure_message` left over.
```suggestion
        self.browser_js('/web/tests/mobile?mod=web', "", "", login='admin', timeout=1800,  error_checker=qunit_error_checker)
```